java-checks-test-sources/src/main/java/checks/spring/ModelAttributeNamingConventionForSpELCheck.java

```diff
@@ -0,0 +1,30 @@
+package checks.spring;
+
+import java.util.Map;
+
+class ModelAttributeNamingConventionForSpELCheck {
+
+  public void foo(org.springframework.ui.Model model) {
+    model.addAllAttributes(Map.of("m", 42, "a", 22)); // Compliant
+    model.addAllAttributes(getMap()); // Compliant
+
+    model.addAllAttributes(Map.of(" m", 42, " a", 22)); // Noncompliant [[sc=35;ec=39]] {{Attribute names must begin with a letter (a-z, A-Z), underscore (_), or dollar sign ($)
```

Comment:
Do we only raise a single issue on this line? Or should we raise on the two problematic elements within the line?